### ISSUE TITLE
NVDA is silent when tab to the first control "Add sign-in method" button in 'Registered security info' region in browse mode.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13736]
### Steps to reproduce:
1. Turn on NVDA and open URL: https://codepen.io/testaccount01/pen/mdXqKvo
2. Ensure NVDA browse mode is enabled by pressing Insert+ Space.
3. Tab from “Change” button to "Add sign-in method" button.
4. Observe that NVDA announce complete region information when region receives focus and remain silent when "Add sign-in method" button receives focus.

### Actual behavior:
NVDA is reading a lot of content when moved from “Change” to the container where the 'Add sign-in method' button and the list of sign-in options are in _“Registered security info region Add sign-in method button separator list Phone Change Phone button….”_
And in next Tab keystroke, it remains silent when tab to the first control "Add sign-in method" button in 'Registered security info' region in browse mode.

### Expected behavior:
"Add sign-in method" control should be announced by NVDA when focused.

### System configuration
Windows 11

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version 2022.1

#### Windows version:
Version 21H2 Build 22000.675

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, NVDA Version 2021.3.5, issue was reproducible with this version as well.

**Comment by seanbudd** [comment]
When tabbing from the change button focus moves to a larger region not to the "Add sign-in method", is this intended?

**Comment by seanbudd** [comment]
This relates to issue #6606

**Comment by Adriani90** [comment]
@msftedad I cannot reproduce the silence of NVDA that you are describing, but I can reproduce the high verbosity when tabbing to the region in browse mode.
Is there a reason why you put tabindex=0 on elements that do not have a role, like plain text and region labels? Tabindex=0 or higher should be put only on elements that have a role and where the user can interact with, like links, buttons etc.
Otherwise you should use tabindex=-1 for those regions labels and plain text descriptions. If you put tabindex=0 only on the buttons, are you still reproducing this issue?

I think this issue is rather related to bad authoring than to NVDA.

cc: @jcsteh in case I am missing something here.

**Comment by seanbudd** [comment]
I am closing this as an authoring issue.
Given there has been no response to my questions from may 2022, we cannot know the expected behaviour given the unusual authoring.
